Caleydo/ordino
# Redesign welcome page and start menu

Part of https://github.com/Caleydo/tdp_bi_bioinfodb/issues/1096

### Summary

* Introduce the new welcome page and start menu
   * Start menu is implemented and discussed here
   * Welcome page is currently implemented and discussed in https://github.com/Caleydo/ordino_public/pull/74
* Use React, React Bootstraps, and Bootstrap 4 for new components
   * There are currently some issues due to BS4 only (see https://github.com/datavisyn/visyn2020/issues/10#issuecomment-752918269 for further information)

### How to test

1. Setup an Ordino Public workspace (`yo phovea:setup-workspace Caleydo/ordino_product -b welcome-page`) or switch your branches accordingly
2. `npm start` and `docker-compose up`
3. Depending on the configured entry points in ordino_public you should either see the welcome page or the Ordino app

### Known Issues

* The styles of the app are broken (see https://github.com/datavisyn/visyn2020/issues/10#issuecomment-752918269 for further information) and only the new start menu, which uses React Bootstrap and BS4, should be displayed correctly

### To dos
- [ ] Update phovea/tdp framework to Bootstrap v4
- [ ] Assess whether we can move some React components of the welcome page from ordino_public to this repository (and encourage reusability over copy+paste to other customizations)
